src/test/silenceErrors.ts

```diff
@@ -4,6 +4,7 @@ const originalConsoleError = console.error;
 const ignoreErrorsList = [
   `Could not parse CSS stylesheet`,
   `Using kebab-case for css properties in objects is not supported.`,
+  `Warning: An update to %s inside a test was not wrapped in act(...).`,
```

Comment:
This started to happen a lot after upgrading deps. It seems to happen in all tests that deal with forms (calling https://github.com/grafana/synthetic-monitoring-app/blob/upgrade-deps/src/components/CheckEditor/testHelpers.ts#L31) but it doesn't seem trivial to fix.